src/ert/field_utils/roff_io.py

```diff
@@ -50,7 +50,7 @@ def export_roff(
 
 def import_roff(
     filelike: Union[TextIO, BinaryIO, _PathLike], name: Optional[str] = None
-) -> np.ma.MaskedArray[Any, np.dtype[np.double]]:
+) -> np.ma.MaskedArray[Any, np.dtype[np.float32]]:
```

Comment:
@JHolba 
Can I assume that the code that does the import does the right thing and loads 32bit floats as that is what is stored in `.roff` files?

Reply:
looks like that should be returning 32bit as long as the file is 32bit and the typing is incorrect currently.
@eivindjahren

Reply:
I've added an explicit conversion to 32bit in case it is 64bit as `xtgeo` can create 64bit `.roff` files.